tests/test_sessions.py

```diff
@@ -83,6 +83,21 @@ def test_properties(self):
         assert session.bin is runner.venv.bin
         assert session.python is runner.func.python
 
+    def test_interactive(self):
+        with mock.patch("nox.sessions.sys") as mock_sys:
```

Comment:
Is it possible to just patch `sys.stdout.istty`?

Reply:
Yep. I just pushed a commit with the change.